Symfony/CS/Tests/Fixer/Contrib/OrderedUseFixerTest.php

```diff
@@ -585,4 +585,644 @@ public function testCodeWithImportsOnly()
 
         $this->makeTest($expected, $input);
     }
+
+    /*
+    |--------------------------------------------------------------------------
+    | Test sorting by length
+    |--------------------------------------------------------------------------
+    */
+
+    /**
+     * @return mixed
+     */
+    protected function sortByLengthFixer()
+    {
+        $fixer = $this->getFixer();
+        $fixer::configure($fixer::SORT_LENGTH);
```

Comment:
after this test state of fixer is changed.
you need to set it up and then teardown, to not introduce side-effects for other tests.

github.com/FriendsOfPHP/PHP-CS-Fixer/blob/1.12/Symfony/CS/Tests/Fixer/Contrib/HeaderCommentFixerTest.php#L30-L41